core/src/test/java/io/questdb/test/griffin/AlterTableAddColumnTest.java

```diff
@@ -94,14 +141,15 @@ public void testAddBadSyntax() throws Exception {
 
     @Test
     public void testAddBusyTable() throws Exception {
+        Assume.assumeFalse(isWal);
```

Comment:
that looks like a lot of effort draining queues just to disable WAL runs like that

Reply:
This line disables only a single test for WAL: `testAddBusyTable()`
The test asserts that `EntryUnavailableException` is thrown when trying to add a column to the table when the writer is locked.
I guess the point of the test is to make sure that `EntryUnavailableException` is thrown, and not something else, because that will put the command on the async queue, instead of rejecting it outright.
This test does not make sense for WAL table.